com.ibm.wala.core/src/test11Subjects/java/privateInterfaceMethods/testArrayReturn/testArrayReturn.java

```diff
@@ -0,0 +1,10 @@
+package privateInterfaceMethods.testArrayReturn;
+
+public class testArrayReturn {
```

Comment:
```suggestion
public class TestArrayReturn {
```

Will have to change elsewhere including `PrivateInterfaceMethodsTest`